examples_src/device-orientation.js

```diff
@@ -31,8 +30,10 @@ var map = new ol.Map({
 });
 
 var deviceOrientation = new ol.DeviceOrientation();
-var track = new ol.dom.Input(document.getElementById('track'));
-track.bindTo('checked', deviceOrientation, 'tracking');
+$('#track').on('change', function() {
+  deviceOrientation.setTracking(
+      /** @type {boolean} */ ($(this).prop('checked')));
```

Comment:
minor but this can be simplified to: `deviceOrientation.setTracking(this.checked);`